x/leverage/simulation/operations.go

```diff
@@ -175,11 +175,15 @@ func SimulateMsgBorrowAsset(ak simulation.AccountKeeper, bk types.BankKeeper, lk
 		r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context,
 		accs []simtypes.Account, chainID string,
 	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
-		from, coin, skip := randomCollateralFields(r, ctx, accs, lk)
+		from, uCoin, skip := randomCollateralFields(r, ctx, accs, lk)
```

Comment:
I think the convention we use is `uToken`, right @toteki?

```suggestion
		from, uToken, skip := randomCollateralFields(r, ctx, accs, lk)
```

Reply:
yeah, usually token and uToken for generic `sdk.Coin` values

Reply:
Update coin to token at https://github.com/umee-network/umee/pull/433/commits/c210c5605b3fe23e67fec59d8b991efc35a91361